mcs/tools/mdoc/Test/en.expected.importecmadoc/System/Action`1.xml

```diff
@@ -21,13 +21,12 @@
     <typeparam name="T">To be added.</typeparam>
     <param name="obj">The object on which to perform an action.</param>
     <summary>
-					<para> Represents the method that performs an action on the specified object.</para>
-				</summary>
+      <para> Represents the method that performs an action on the specified object.</para>
+    </summary>
```

Comment:
Is it? just out of curiosity could you point out where you see this in the build spew (in the interest of me learning how to interpret the results better)? 

That being said, I don't remember making this change explicitly ... I rebased my branch *after* that commit was merged into master that we talked about via email (the one with the xml formatter changed).